### ISSUE TITLE
Less than (<, &lt;) is being escaped and generates strange output

### ISSUE THREAD

**Issue by dellagustin** [rmtiwari/ABAPSyntaxHighlighter#4]
![image](https://cloud.githubusercontent.com/assets/260113/8551538/91d0c248-24d7-11e5-94e8-6ec8097aa1a9.png)

When the text has a less then character (<), it is shown as its escaped version (&lt;).

The reason seems to be prismABAP.js line 137:

``` javascript
code = code.replace(/&/g, '&amp;').replace(/</g, '&lt;').replace(/\u00a0/g, ' ');
```

Why is this here? 
I compared with the current version of prism.js and there it is:

``` javascript
code = code.replace(/^(?:\r?\n|\r)/,'');
```

**Comment by rmtiwari** [comment]
..looks like there is a breaking change in new version of prism - above change is from below
https://github.com/rip-lang/prism/commit/c0a0353de48cd19ab53c0d719d504a74666f8495

**Comment by dellagustin** [comment]
What about forking https://github.com/PrismJS/prism and introducing ABAP as an officially supported language instead of keeping a parallel project?

**Comment by dellagustin** [comment]
Will be solved togheter with #5

**Comment by rmtiwari** [comment]
..just to add that not sure about latest version but earlier prism core expected the code inside PRE tag to be already escaped. Actually it's using textContent and _not_ innerHTML ( escaped to avoid XSS ) so the referenced code to replace the < to &lt; will not work - no idea why it was there in prism core...figured this out and then also found this tweet https://twitter.com/xon1c/status/316206379864039424

**Comment by dellagustin** [comment]
closed togheter with #5.
solved in 05b157e4bfcf49bd13908d98faf348a891a111ab
